src/fix_langevin.cpp

```diff
@@ -260,9 +287,14 @@ void FixLangevin::init()
   if (!atom->rmass) {
     for (int i = 1; i <= atom->ntypes; i++) {
       gfactor1[i] = -atom->mass[i] / t_period / force->ftm2v;
-      gfactor2[i] = sqrt(atom->mass[i]) *
-        sqrt(24.0*force->boltz/t_period/update->dt/force->mvv2e) /
-        force->ftm2v;
+      if (!gjfflag)
```

Comment:
this would be easier to read, if the test was `if (gjfflag)` and the cases swapped.